rocky/views/upload_csv.py

```diff
@@ -56,17 +59,24 @@ def get_context_data(self, **kwargs):
         context["criterias"] = CSV_CRITERIAS
         return context
 
-    def get_ooi_from_csv(self, ooi_type: str, value: str):
-        ooi = None
+    def get_network(self, network: str):
+        if "network" in self.networks:
+            return self.networks[network]
+        network = Network(name=values["network"])
+        self.networks[network] = network
+        return network
+
+    def get_ooi_from_csv(self, ooi_type: str, values: dict[str, str]):
+        network = self.get_network(values["network"] if "network" in values.keys() else "internet")
```

Comment:
```suggestion
        network = self.get_network(values.get("network", "internet"))
```

Reply:
agreed, and merged with the previous suggestion